Resources/Prototypes/Entities/Weapons/Pistols/pistols.yml

```diff
@@ -19,7 +19,7 @@
     sound_gunshot: /Audio/Guns/Gunshots/pistol.ogg
 
 - type: entity
-  name: Clarissa
+  name: clarissa
```

Comment:
I think these should be capitalized, since they're actual gun *nicknames*? Hell there's Deckard down there which is a plain Blade Runner reference.